src/pages/cluster/ClusterMemberMemoryUsage.tsx

```diff
@@ -0,0 +1,76 @@
+import type { FC } from "react";
+import { useQuery } from "@tanstack/react-query";
+import Meter from "components/Meter";
+import { fetchResources } from "api/server";
+import { fetchClusterMemberState } from "api/cluster-members";
+import { queryKeys } from "util/queryKeys";
+import { humanFileSize } from "util/helpers";
+import type { LxdClusterMember } from "types/cluster";
+
+interface Props {
+  member: LxdClusterMember;
+}
+
+const ClusterMemberMemoryUsage: FC<Props> = ({ member }) => {
+  const { data: resources } = useQuery({
+    queryKey: [
+      queryKeys.cluster,
+      queryKeys.members,
+      member.server_name,
+      queryKeys.resources,
+    ],
+    queryFn: async () => fetchResources(member.server_name),
+  });
+
+  const { data: state } = useQuery({
+    queryKey: [
+      queryKeys.cluster,
+      queryKeys.members,
+      member.server_name,
+      queryKeys.state,
+    ],
+    queryFn: async () => fetchClusterMemberState(member.server_name),
+  });
```

Comment:
This is rather expensive to fetch for each member the resources *and* state. Maybe we avoid this call and remove the hover text to keep the overhead smaller?